plugins/aea-cli-benchmark/pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
 requires = ["setuptools", "wheel"]
-build_backend = "setuptools.build_meta"
+build-backend = "setuptools.build_meta"
```

Comment:
@8ball030 Can you confirm that these changes are compatible with the build tools we use? I'm assuming these were incompatible with the poetry build tools on your fork

Reply:
I ran the pythons setup build etc that you guys run.

Please run locally to confirm

Reply:
AFAIC i ran exactly what is in the release workflow i contributed to